packages/text-annotator/package.json

```diff
@@ -40,7 +40,9 @@
     "@annotorious/core": "^3.0.0-rc.30",
     "colord": "^2.9.3",
     "dequal": "^2.0.3",
+    "hotkeys-js": "^3.13.7",
     "rbush": "^4.0.0",
+    "ts-key-enum": "^2.0.12",
     "uuid": "^10.0.0"
```

Comment:
I need to review in detail later. But one Q in advance: did you check what the impact is on these dependencies on the overall bundle size? I feel that with basic things, there's always a bit of cost/benefit challenge.

Reply:
Fortunately, the `ts-key-enum` is implemented as the `const enum` which makes the `Key.Smth` calls replaced w/ real value during the compilation. Therefore, the cost of using the `ts-key-enum` is almost negligible ([source](https://www.npmjs.com/package/ts-key-enum#v3))

> This allows the enum's definition to live entirely in the definition file `https://gitlab.com/nfriend/ts-key-enum/blob/master/Key.enum.d.ts`. Consumers can use this enum without including all of the enum's values in their own JavaScript bundle; only the values referenced in their code will be injected into their bundle.

Reply:
> `hotkey-js`

> has a reasonable footprint ([~6kB](https://bundlephobia.com/result?p=hotkeys-js)) (gzipped: 2.8kB), and has no dependencies. It should not interfere with any JavaScript libraries or frameworks